docs/src/app/components/pages/components/DatePicker/ExampleInline.js

```diff
@@ -8,6 +8,12 @@ const DatePickerExampleInline = () => (
   <div>
     <DatePicker hintText="Portrait Inline Dialog" container="inline" />
     <DatePicker hintText="Landscape Inline Dialog" container="inline" mode="landscape" />
+    <DatePicker
+      hintText="Keyboard Enabled Dialog"
+      container="inline"
+      mode="landscape"
+      keyboardEnabled={true}
```

Comment:
@nicolaiskogheim propType declarations are definitely alphabetical. I don't know about prop assignments, but either way this is just the docs file and the previous examples don't follow this rule either.